### PR TITLE
Verify that proto-generated files are up-to-date

### LINKED ISSUE #1431: Ensure that proto/grpc dependencies are in sync with generated files
We don't have a validation step to ensure that the proto-generated source code is consistent with the versions of declared dependencies. For example, as of 8e76a50 (Mar 19 2019), there are the following differences:
```
$ dep ensure
$ make proto-install
$ make proto
$ git status
	modified:   proto-gen/api_v2/api_v2.pb.gw.go
	modified:   proto-gen/openapi/api_v2.swagger.json
```

### PR DESCRIPTION
**Pull request by yurishkuro**
Resolves #1431

### CODE REVIEW

**Comment by yurishkuro** [comment]
Verification that the test works: https://travis-ci.org/jaegertracing/jaeger/jobs/508563896#L626

image

**Comment by yurishkuro** [comment]
Crossdock test came back green on re-run. Booked #1434 for that.

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ matrix:
     env:
     - TESTS=true
     - COVERAGE=true
+  - go: "1.11.1"
```
Comment (black-adder) [inline .travis.yml]: at this rate we're going to hit the 1 hour mark for CI, we should do some research into smarter CI or combine some steps into 1.

File: Makefile [hunk 2]
```diff
@@ -397,5 +398,4 @@ proto-install:
 		./vendor/github.com/gogo/protobuf/protoc-gen-gogo \
 		./vendor/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-grpc-gateway \
 		./vendor/github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger
-		# ./vendor/github.com/mwitkow/go-proto-validators/protoc-gen-govalidators \
-		# ./vendor/github.com/rakyll/statik
+		# ./vendor/github.com/mwitkow/go-proto-validators/protoc-gen-govalidators
```
Comment (black-adder) [inline Makefile]: can we remove if it's commented out?

File: .travis.yml (hunk 1 above)
Comment (yurishkuro) [reply .travis.yml]: most steps = more parallelization. The real issue is that Travis is throttling us on the number of concurrent steps.

**Pull request merged**